packages/db/scripts/cmds/db-sql.js

```diff
@@ -64,29 +45,56 @@ export async function dbSqlCmd ({ reset, cargo, testing } = {}) {
   await client.query(configSql)
   await client.query(tablesSql)
 
-  if (cargo) {
-    if (testing) {
-      cargoSql = cargoSql.replace(
-        `
--- Create materialized view from cargo "aggregate_entries" table
-CREATE MATERIALIZED VIEW public.aggregate_entry
-AS
-SELECT *
-FROM cargo.aggregate_entries;`,
-        `
-CREATE MATERIALIZED VIEW public.aggregate_entry
-AS
-SELECT *
-FROM cargo.aggregate_entries 
-WHERE cid_v1 in ('bafybeiaj5yqocsg5cxsuhtvclnh4ulmrgsmnfbhbrfxrc3u2kkh35mts4e');
-`
-      )
-    }
-
+  // if testing or cargo fdw flag not set, you just get the schema, no fdw connection to dagcargo
+  if (testing || !cargo) {
+    await client.query(cargoSchema)
+  } else {
```

Comment:
On second thoughts I think it makes more sense to have cargo stuff only when you pass the cargo flag:

```suggestion
  // if testing or cargo fdw flag not set, you just get the schema, no fdw connection to dagcargo
  if (testing && cargo) {
    await client.query(cargoSchema)
  } else if (!testing && cargo) {
```